internal/testing/e2e/in_memory.go

```diff
@@ -190,9 +190,9 @@ api:
   scale:
     httpport: "51509"
   evaluator:
-    hostname: "om-evaluator"
-    grpcport: "50508"
-    httpport: "51508"
+    hostname: "test"
```

Comment:
I think you overwrite the evaluator values at L138 so changing the values at here wont take any effects

Reply:
I want this to match exactly that the real cluster's config looks like, will add test which ensures that in a future PR.  (it's still wrong, but I want changing it there, so I wanted to change it here.)

Reply:
idk.. but having the in cluster config being override in the Makefile and the in memory one being set in a go file looks pretty weird @_@